insights/parsers/fc_match.py

```diff
@@ -0,0 +1,54 @@
+"""
+/usr/bin/fc-match -f 'sans:regular:roman' -sv | grep -e 'family:' -e 'fontformat' - Command
+===========================================================================================
+
+This command gets the fonts information in the current system.
+
+Typical output of this command is::
+
+    family: "DejaVu Sans"(s)
+    fontformat: "TrueType"(s)
+    family: "DejaVu Sans"(s)
+    fontformat: "TrueType"(s)
+    family: "DejaVu Sans"(s)
+    fontformat: "TrueType"(s)
+    family: "DejaVu Sans"(s)
+    fontformat: "TrueType"(s)
+    family: "Nimbus Sans L"(s)
+    fontformat: "Type 1"(s)
+    family: "Standard Symbols L"(s)
+    fontformat: "Type 1"(s)
+
+Examples:
+    >>> fc_match = shared[FCMatch]
+    >>> fc_match_info = fc_match.fonts_info
+    >>> fc_match_info
+    [{'fontformat': '"TrueType"(s)', 'family': '"DejaVu Sans"(s)'},{'fontformat': '"Type 1"(s)', 'family': '"Nimbus Sans L"(s)'}, {'fontformat': '"Type 1"(s)', 'family': '"Standard Symbols L"(s)'}]
+"""
+
+from .. import Parser, parser
+
+
+@parser("fc-match")
+class FCMatch(Parser):
+    """Class to parse command `/usr/bin/fc-match -f 'sans:regular:roman' -sv | grep -e 'family:' -e 'fontformat'`."""
+
+    def parse_content(self, content):
+        self.data = []
+        font = {}
+        for l in content:
+            k, v = l.strip().split(":")
+            if k == "family":
+                font["family"] = v.strip()
+            else:
+                font["fontformat"] = v.strip()
+                if font not in self.data:
+                    self.data.append(font)
+                    font = {}
+
+    @property
```

Comment:
Instead of this one, better add a `__iter__`  interface to `yield` each item in the list and a `__getitem__` to support `[]` index.